versioned_docs/version-1.3.0/release-notes/release-notes-for-kubeslice-oss-1.3.1.mdx

```diff
@@ -11,7 +11,7 @@ We continue to add new features and enhancements to KubeSlice.
 
 ## What's New :speaker:
 
-Installing KubeSlice in the Air-Gapped Environment using the kubeslice-cli Tool
+## Installing KubeSlice in the Air-Gapped Environment using the kubeslice-cli Tool
```

Comment:
This should be third-level heading ### 

What's New is second level heading - ##

Reply:
done